### ISSUE TITLE
EWPE SMART

### ISSUE THREAD

**Issue by sensnowy** [norzechowicz/homebridge-ch-ac#1]
Since this work using ewpa smart, do you think I could use it to control aermec / novatherm units ? 
Thank you !

**Comment by norberttech** [comment]
Hey, hard to say honestly, I forked it some time ago and never touched it since then, as a matter of fact I'm not using it now but since hot months are coming and I just replaced my NAS solution I might need to reactivate it. Meaning that I might want to rewrite it from scratch to make it more stable

**Comment by sensnowy** [comment]
That would be cool. I am just getting a new AC installed and they all got these ewpa smart units in them. Looking at the code it’s just the IP address the code is access. 
By chance are you German ? Sorry for asking.

**Comment by norberttech** [comment]
yeah those units are not secured at all, unfortunately - it's better to keep them in isolated subnetwork with one way communication.   

> By chance are you German ? Sorry for asking.

No sorry, I'm from Poland

**Comment by mountbatt** [comment]
@sensnowy Hi, ich spreche Deutsch und habe demnächst eine Sinclair Anlage am Start und suche ebenso eine Einbindung in Homebridge. Das eingebaute WLAN Modul wird ebenso mit dieser EWPE Smart App realisiert. Vielleicht finden wir ja gemeinsam eine Lösung. – Alternativ wäre noch was mit einem Infrarot-Dongle machbar (Broadlink RM Mini). Dieser kann in die Fernbedienung mit Infrarot simulieren und ist auch über Homebridge steuerbar. Hast Du schon irgendwas erreicht?

**Comment by sensnowy** [comment]
Ich hab von tado für 55€ das gekauft. War im Angebot und mega einfach.

**Comment by mountbatt** [comment]
@sensnowy ok! Welche Version?

**Comment by mountbatt** [comment]
@sensnowy ah mit tado Code TADO44 😀

**Comment by sensnowy** [comment]
Ja
